helper/astutils/ident.go

```diff
@@ -0,0 +1,24 @@
+package astutils
+
+import "go/ast"
+
+// IdentExprStringValue fetches a string value from the Expr
+// If the Expr is not Ident, returns an empty string.
+func IdentExprStringValue(e ast.Expr) *string {
+	switch v := e.(type) {
+	case *ast.Ident:
+		if v.Obj != nil && v.Obj.Decl != nil {
+			switch v.Obj.Decl.(type) {
+			case *ast.AssignStmt:
+				if assignStmt := v.Obj.Decl.(*ast.AssignStmt); assignStmt != nil && len(assignStmt.Rhs) != 0 {
+					return ExprStringValue(assignStmt.Rhs[0])
+				}
+			case *ast.ValueSpec:
+				if valueSpec := v.Obj.Decl.(*ast.ValueSpec); valueSpec != nil && len(valueSpec.Values) != 0 {
+					return ExprStringValue(valueSpec.Values[0])
+				}
+			}
+		}
+	}
+	return nil
+}
```

Comment:
This is getting into a gray area of working with the AST since the context is ambiguous depending on the source code. We typically want to avoid making any assumptions about the source code where possible unless a style is enforced somehow during compilation. For example, choosing the first assignment statement type value (`Rhs[0]`) is potentially not accurate in the case of this valid code:

```go
myVar1, myVar2 := "one", "two"
```

I think in the future if we have a use case for needing actual values, we would prefer not working in the AST at all and come up with a more tailored solution involving unit testing of the compiled code, allowing the runtime to fill in values properly. 👍 

```suggestion
```

Reply:
Ahh true! I was playing around a bit much with the AST 😅  a more generic way of testing would be awesome 👍